queries.sql

```diff
@@ -33,3 +33,63 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 -----------------------------------------------
 -----------------------------------------------
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] Good job so far, :clap: however, according to the project requirement you should `Verify that change was made` before COMMIT just like what you did after COMMIT by using a select statement. Please resolve that :slightly_smiling_face: